pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -54,6 +54,12 @@ public enum Strategy {
   @JsonPropertyDescription("Column for upsert comparison, default to time column")
   private String _comparisonColumn;
 
+  @JsonPropertyDescription("Custom class for upsert metadata manager")
+  private String _metadataManagerClass;
+
+  @JsonPropertyDescription("Custom properties")
+  private Map<String, String> _properties;
```

Comment:
I think we should rename it to `_metadataManagerConfigs` or `_configs` from `_properties`. Currently we do it in a similar way for record reader and filesystems.

Reply:
Done